### DISCUSSION TITLE
Why ICS import can't be imported to any other provider (Proton etc)

### DISCUSSION THREAD

**Discussion by abuturabofficial** [FossifyOrg/General-Discussion#476] [General]
I tried to import my ICS file to proton but got an error saying:

```log
Proton Calendar currently does not support certain event types, details, and formats.
````
![image](https://github.com/FossifyOrg/Calendar/assets/68259771/55c9dc48-5f1d-4191-aa42-485ec77e2006)

I even tried on Google Calendar (just to test, it's proton only problem), but it didn't work either.

What's going on? Why is ICS file not universal?

**Comment by Aga-C** [comment]
Does your ICS file contain tasks? If so, Google Calendar doesn't support importing tasks from an ICS file, [despite them being part of the standard](https://icalendar.org/iCalendar-RFC-5545/3-6-2-to-do-component.html). I don't have Proton, but seeing your error, I assume it also doesn't support tasks.

**Reply by abuturabofficial** [reply]
Yes, I do have tasks.
